docs/en/week10/10-3.md

```diff
@@ -0,0 +1,85 @@
+---
+lang-ref: ch.10-3
+lecturer: Alfredo Canziani
+title: Transformer  Encoder-predictor-decoder architecture
+authors: Rahul Ahuja, jingshuai jiang 
+date: 15th April 2021
+---
+
+
+## The Transformer
+
+Before elaborating the encoder-predictor-decoder architecture, we are going to review two models we've seen before.
+
+
+### Conditional EBM latent variable architecture
+
+
+We should be familiar with the terminology of these modules from the previous lectures. 
+In the conditional EBM latent variable architecture, we have $\vect{x}$ the conditional variable which goes into a predictor. We have $\vect{y}$ which is the target value. The decoder modules will produce $\tilde{\vect{y}}$ when fed with a latent variable $\vect{z}$ and the output of the predictor. $E$ is the energy function which minimizes the energy between $\tilde{\vect{y}}$ and $\vect{y}$. 
+
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/ebm.png" style="zoom: 80%; background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> Conditional EBM Model.
+</center>
+
+### Autoencoder architecture
+
+In Autoencoder architecture , we observed there is no conditional input but only a target variable. The entire architecture is trying to learn the structure in these target variables. The target value $\vect{y}$ is fed through an encoder module which transforms into a hidden representation space, forcing only the most important information through. And the decoder will make these variables come back to the original target space with a $\tilde{\vect{y}}$. And the cost function will try to minimize the distance between $\tilde{\vect{y}}$ and $\vect{y}$. 
+
+
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/autoencoder.png" style="zoom: 80%; background-color:#DCDCDC;" /><br>
+<b>Figure 2:</b> Autoencoder Architecture.
+</center>
+
+
+
+### Encoder-predictor-decoder architecture
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/transformer.png" style="zoom: 80%; background-color:#DCDCDC;" /><br>
+<b>Figure 3:</b> Transformer Architecture.
+</center>
+
+
+In a transformer, $\vect{y}$ (target sentence) is a discrete time signal. It has discrete representation in a time index. The $\vect{y}$ is fed into a unit delay module succeeded by an encoder. The unit delay here transforms $\vect{y}[j] \rightarrow \vect{y}[j-1]$. The only difference with the autoencoder here is this delayed variable. So we can use this structure in the language model to produce the future when given the past.
```

Comment:
I think you want to use `\mapsto` symbol. `\to` is used for limits.

Reply:
fixed